app/services/construct_cap_attributes.rb

```diff
@@ -41,33 +39,18 @@ def product_name_and_description
       #{auction.summary})
   end
 
-  def link_to_product
-    auction.delivery_url
-  end
-
   def justification
     "Tock line item: #{auction.billable_to}"
   end
 
   def cost_per_unit
-    winning_bid.amount
+    auction.start_price
```

Comment:
yeah, this is part of the functionality change we needed. The highest price needs to be included for approval before the auction runs, but the proposal is updated with real bid amount for purchase step on C2. C2 tracks all changes (a comment is added to the proposal after each update) so the changes will be recorded on their side.